### PR TITLE
add script to install ingress autoscaling

### PR DESCRIPTION
**Pull request by qiliRedHat**

### CODE REVIEW

File: openshift_scalability/install-ingress-autoscaling.sh [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+    echo "[ERROR] please make sure KedaController is successfully installed."
+    exit 1
+fi
+echo "[INFO ]====Installing Ingresss Autoscaling===="
+
+echo "[INFO] Creating project openshift-ingress-operator"
+oc project openshift-ingress-operator
+
+echo "[INFO] Applying ConfigMap cluster-monitoring-config"
+oc apply -f -<<EOF
```
Comment (paigerube14) [inline openshift_scalability/install-ingress-autoscaling.sh:32]: Let's add all these yaml files to the content folder or in a different folder in this directory to be reusable

**Comment by qiliRedHat** [comment]
@paigerube14 PTAL

File: openshift_scalability/scripts/ingress_autoscaling/install-ingress-autoscaling.sh [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+fi
+oc apply -f content/ingress-scaler.yaml
+
+echo "[INFO] ====Post Install Check===="
+echo "[INFO] Getting hpa"
+oc get hpa
+echo "[INFO] Getting ingresscontroller replicas"
+oc get ingresscontroller/default -o yaml -n openshift-ingress-operator | grep replicas:
+echo "[INFO] Getting number of worker nodes:"
+oc get nodes --no-headers -l node-role.kubernetes.io/worker= | wc -l
```
Comment (paigerube14) [inline openshift_scalability/scripts/ingress_autoscaling/install-ingress-autoscaling.sh:85]: I think there should be some validation here? I'm not 100% sure what should be the result. I was able to properly run with the help of the comments and the validation you added above :)

Comment (qiliRedHat) [reply openshift_scalability/scripts/ingress_autoscaling/install-ingress-autoscaling.sh:85]: This is a script to help enabling the feature before running the actual test cases. I haven't automate the test cases yet. But you remind me it's also good to add part of the validations here so I don't need to do them again on the test case side.

**Comment by qiliRedHat** [comment]
@paigerube14 PTAL

**Comment by qiliRedHat** [comment]
Merged after getting lgtm in peer review.

**Pull request merged**